exercises/practice/pov/pov.go

```diff
@@ -1,23 +1,52 @@
 package pov
 
-// Define the Graph type here.
+// Define the Tree type here
 
-func New() *Graph {
-	panic("Please implement the New function")
+type Tree struct{}
+
+// New creates and returns a new Tree with the given root value and children
+// You can assume that there will be no duplicate values in test trees.
```

Comment:
Line 8 should go into the instructions (append), not into the function comment.

Reply:
Done :)